src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  color: $main-color;
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    line-height: 18px;
+    margin-bottom: 4px;
+    font-weight: 500;
+    width: 100%;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__discription {
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+    width: 100%;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: 100%;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 16px;
+    width: 100%;
+
+    &-text {
+      line-height: 18px;
+      color: $secondary-color;
+    }
+
+    &-amount {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__buy-link {
+    display: flex;
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: #fff;
+    border: 1px solid $button-color;
+    background: $button-color;
+
+    &:hover {
+      color: $button-color;
+      background-color: #fff;
+    }
```

Comment:
Add transition for these properties too for the smooth changes.
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/10d06224-5faf-4b90-9397-38add322c10d)